### ISSUE TITLE
v8: Select Featured Products macro should use product picker

### ISSUE THREAD

**Issue by bjarnef** [umbraco/The-Starter-Kit#63]
In starterkit for Umbraco 8 it has a macro "Select Featured Products" (which can be inserted in grid). This macro has a parameter "Choose product" which allow selecting any node.

![image](https://user-images.githubusercontent.com/2919859/53957047-a6173680-40dd-11e9-9c65-a09be81d943b.png)

It would be great if it could use a product picker like the following (based on MNTP), but unfortunately the macro parameter doesn't allow specifying configuration of (macro) property editors.

![image](https://user-images.githubusercontent.com/2919859/53957175-f393a380-40dd-11e9-9fa0-8fbc27a1ec20.png)

**Comment by nul800sebastiaan** [comment]
@bjarnef What is the request? Being able to use MNTP to pick stuff into macros?

**Comment by bjarnef** [comment]
@nul800sebastiaan yes, as the current MNTP product picker, but macros parameter editor don't have configuration.

**Comment by lauraneto** [comment]
Closing this as macros (and grid) are obsolete (removed in v14+).
